draft-ietf-quic-recovery.md

```diff
@@ -166,12 +157,50 @@ measure and report the delay from when a packet was received by the OS kernel,
 which is useful in receivers which may incur delays such as context-switch
 latency before a userspace QUIC receiver processes a received packet.
 
+
 # Loss Detection
 
-We now describe QUIC's loss detection as functions that should be called on
-packet transmission, when a packet is acked, and timer expiration events.
+## Overview {#overview}
```

Comment:
You don't need a specific overview section here.  The overview can go in the top-level section.  My technical writing teacher say that it is bad form to include empty sections, even if they have child sections, so you can avoid two problems.  Also, I assume that you need an overview for congestion control as well, so you avoid creating two sections with the label "overview".